web_widget_uom_view_precision/models/uom_view_precision_backend.py

```diff
@@ -0,0 +1,19 @@
+from odoo import api, models
+
+
+class UomViewPrecisionBackend(models.AbstractModel):
+
+    _name = 'uom.view.precision.backend'
+    _description = 'Uom View Precision Backend'
+
+    @api.model
+    def get_uom_view_precisions(self):
```

Comment:
I don't see the interest of this abstract class. why not just move this function in the ``uom_uom.py`` file ?